src/schemas/Certificates.ts

```diff
@@ -8,14 +8,14 @@ export interface CertificatesSchema {
 	signerKeyPassphrase?: string;
 }
 
-// Joi.binary is not available in the browser
-const maybeBinaryOrString = Joi.binary ? [Joi.binary(), Joi.string()] : [Joi.string()];
+// Joi.binary is not available in the browser so fallback to basic check
```

Comment:
Since cloudflare workers are kind of a hybrid of browser and node.js environments, I think this comment might bring people confusion, since we don't actually support browsers. CW are "weird way" of doing serverless functions.

I'd write 

> Joi.binary is not available in browser-like environments (like Cloudflare workers) ...

What do you think about it? 😄

Reply:
Yeah sounds good!